genhub/genomes/Cbir.yml

```diff
@@ -1,14 +1,10 @@
 Cbir:
-    species: Cerapachys biroi
+    species: Ooceraea biroi
```

Comment:
If possible, let's maintain this original recipe when we add the new one. Like honeybee, this will require updating this recipe to point to GenBank.

Reply:
I understand the desire to do this with Amel, where the genome was updated, but is there a need to do so here, where the name itself has been changed, as opposed to the version number?